.github/workflows/cron.yml

```diff
@@ -32,6 +32,9 @@ jobs:
         git add .
         git commit -m "Add daily data"
     - name: Push changes
-      uses: ad-m/github-push-action@master
+      run: git push origin master
```

Comment:
Turns out git commands are authenticated when using v2 of the `checkout` action. No need for a 3rd party for a simple Git command

Reply:
Maybe I'm lost in the diff, but where is the "No need for a 3rd party for a simple Git command" part gone?

Reply:
Right on this line. Instead of using a 3rd party GitHub action, I switched to a Git command

Reply:
🙈